config/redirects.js

```diff
@@ -3710,42 +3717,46 @@ const redirects = [
       '/brand-and-customize/customize-login-text-prompts/prompt-signup-password',
       '/customize/universal-login-pages/customize-login-text-prompts/prompt-signup-password',
     ],
-    to: '/customize/universal-login-pages/customize-login-text-prompts',
+    to: '/customize/login-pages/universal-login/customize-text-elements',
   },
   {
     from: [
       '/scopes/customize-consent-prompts',
       '/scopes/current/guides/customize-consent-prompt',
       '/brand-and-customize/customize-consent-prompts',
+      '/customize/universal-login-pages/customize-consent-prompts',
     ],
-    to: '/customize/universal-login-pages/customize-consent-prompts',
+    to: '/customize/login-pages/customize-consent-prompts',
```

Comment:
Quick question here because I haven't seen the new nav for UL - is this page supposed to be under /universal-login or /classic-login or is this a both kind of section?